pkg/secretfilter/logfilter.go

```diff
@@ -0,0 +1,82 @@
+package secretfilter
```

Comment:
Should we rename this file to `secretfilter.go` and the test as well?

Reply:
Yep! done in https://github.com/BlakeWilliams/viewproxy/pull/52/commits/540681bbd41186bbb7c447842456c31121524f68